visual-tests/tests/change-rows-order.spec.js

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+import { helpers } from '../imports/helpers';
+
+const stylesToAdd = [
+  helpers.cssFiles.cookieInfo,
+  helpers.cssFiles.dynamicDataFreeze
+];
+
+test(helpers.testTitle(path.basename(__filename)), async({ page }, workerInfo) => {
+  helpers.init(workerInfo);
+  await page.goto(helpers.testURL);
+  await expect(page).toHaveTitle(helpers.expectedPageTitle);
+  stylesToAdd.forEach(item => page.addStyleTag({ path: helpers.cssPath(item) }));
+
+  const table = page.locator(helpers.selectors.mainTable);
+
+  await table.waitFor();
+  helpers.mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
+  helpers.tbody = table.locator(helpers.selectors.mainTableBody);
+  helpers.thead = table.locator(helpers.selectors.mainTableHead);
```

Comment:
```suggestion
  await table.waitFor();

  const mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
  const cell = mainTableFirstColumn.locator(helpers.findCell({ row: 4, cell: 1, cellType: 'th' }));
```

Reply:
The same thing appears in few files - it was done intentionally as a compromise to make tester's life easier, now should be modified as we discussed.